framework/src/mysql.js

```diff
@@ -13,15 +13,19 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+const LoggerConfig = require('./logger').init;
 const Logger = require('./logger').get;
+const config = require('./config');
 
-const logger = Logger();
+LoggerConfig(config.log);
```

Comment:
Instead of setting the LoggerConfig here, can we add the default config to logger.js so that the logger works by default even if it is not configured in an external application.